Jasmine/Common/ViewModels/TimedViewModelProtocol.swift

```diff
@@ -0,0 +1,21 @@
+import Foundation
+
+/// A ViewModel that implements a timer, i.e. it is timed.
+protocol TimedViewModelProtocol: BaseViewModelProtocol {
```

Comment:
The base VM protocol needs to have `updateGameStatus`, `UpdateTime` etc methods for the timer to call.
Then in your specific VM you have to implement these methods, and in your implementation you could call your specific VC's methods?